ext/cl_khr_external_semaphore.asciidoc

```diff
@@ -317,6 +327,36 @@ imported payload, the semaphore's prior permanent payload will be restored.
 Please refer to handle specific specifications for more details on transference and
 permanence requirements specific to handle type.
 
+To import a handle of type _handle_type_ into an existing semaphore, call the function
+
+include::{generated}/api/protos/clImportSemaphoreKHR.txt[]
+
+_sema_object_ specifies a valid semaphore object with importable properties.
+
+_import_props_  Must be `NULL`.  Reserved for future use.
+
+_handle_type_ specifies the type of semaphore handle being imported, and must be one of the
+handle types imported when _sema_object_ was created.
+
+_handle_size_ specifies the size of memory pointed by _handle_ptr_.
+
+_handle_ptr_ is a pointer to the imported external handle.
+If _handle_ptr_ is `NULL`, _handle_size_ret_ must not be `NULL`.
+
+_handle_size_ret_ returns the actual size in bytes for the external handle.
+If _handle_size_ret_ is `NULL`, it is ignored.
+
+Calling {clImportSemaphoreKHR} is equivalent to destroying _sema_object_ and re-creating it with the original _sema_props_
```

Comment:
Are implementations strictly required to destroy sema_object and recreate it by associating with new handle? Or just closing earlier handle reference and associating new handle reference with sema_object good enough?